### PR TITLE
[13.0] [IMP] fieldservice: Minimize partner_fax dependency

### PR DESCRIPTION
**Pull request by brian10048**
#674 

Added a separate module to retain the partner_fax feature and eliminated the dependency from the base fieldservice module

### CODE REVIEW

**Review by hparfr — APPROVED** [review]

File: fieldservice_partner_fax/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+{
+    "name": "Field Service - Partner Fax",
+    "summary": "This module allows you to manage a partner's fax number"
+    " from the field service location screen.",
+    "version": "13.0.1.0.0",
+    "category": "Hidden",
+    "author": "Open Source Integrators, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/field-service",
+    "depends": ["fieldservice, partner_fax"],
```
Comment (hparfr) [inline fieldservice_partner_fax/__manifest__.py]:
```suggestion
    "depends": ["fieldservice", "partner_fax"],
```

**Review by hparfr — CHANGES_REQUESTED** [review]

File: fieldservice_partner_fax/__manifest__.py (hunk 1 above)
Comment (brian10048) [reply fieldservice_partner_fax/__manifest__.py]: 👍

**Review by bodedra — APPROVED** [review]
Functional review. :+1:

File: fieldservice_partner_fax/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2021 Brian McMaster
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Field Service - Partner Fax",
+    "summary": "This module allows you to manage a partner's fax number"
+    " from the field service location screen.",
+    "version": "13.0.1.0.0",
+    "category": "Hidden",
```
Comment (bodedra) [inline fieldservice_partner_fax/__manifest__.py]:
```suggestion
    "category": "Field Service",
```

**Pull request merged**